src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+public class Ball {
+    private Color color;
+    private int number;
+
+    public Ball(Color color, int number) {
+        this.color = color;
+        this.number = number;
+    }
+
+    @Override
+    public String toString() {
```

Comment:
there are no requirement for custom toString, so please use default one (right click -> gererate -> toString)